Tools/cases_generator/parser.py

```diff
@@ -0,0 +1,72 @@
+from parsing import (
+    InstDef,
+    Macro,
+    Pseudo,
+    Family,
+    Parser,
+    Context,
+    CacheEffect,
+    StackEffect,
+    OpName,
+    AstNode,
+)
+from formatting import prettify_filename
+
+
+BEGIN_MARKER = "// BEGIN BYTECODES //"
+END_MARKER = "// END BYTECODES //"
+
+
+Context = Context
```

Comment:
why is this needed?

Reply:
I was trying to convince MyPy not to complain. Apparently the way to do that is to define `__all__`.
I'll remove it.

https://github.com/python/mypy/issues/16596